pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -65,6 +65,7 @@
   SEGMENT_DOWNLOAD_FAILURES("segments", false),
   NUM_RESIZES("numResizes", false),
   NO_TABLE_ACCESS("tables", true),
+  FAILED_INDEX_VALUES("attribute values", true),
```

Comment:
Rename to `INDEX_VALUE_FAILURES` and change unit to camel case to be consistent with other meters?

Reply:
I'm also not sure how the metrics are emitted underlying, so it is safer to just follow whatever we have now :-P